src/main/kotlin/blackjack/domain/Participant.kt

```diff
@@ -0,0 +1,25 @@
+package blackjack.domain
+
+import blackjack.domain.enums.Condition
+import blackjack.domain.enums.MatchResult
+
+open class Participant(
+    val name: String,
+    val cards: Cards,
+    protected open val condition: Condition
+) {
+
```

Comment:
제가 의도했던 건 condition을 var로 선언하되 protected setter를 지정해주는 것이였습니다😅
이렇게하면 상속한 타입에서 condition을 별도로 오버라이딩하지 않고도 condition을 클래스 내부에서만 변경할 수 있습니다.
```suggestion
open class Participant(
    val name: String,
    val cards: Cards
    condition: Condition
) {
    var condition: Condition = condition
        protected set
}
```

추가로 Participant는 인스턴스로 생성되지 않으니 abstract class로 선언하면 어떨까요?

Reply:
제가 잘못 생각했었군요..!! 해당 방법 참고하겠습니다!!
추상 클래스로도 선언 변경하였습니다~!!